browser/ui/webui/new_tab_page/brave_new_tab_message_handler.cc

```diff
@@ -371,15 +371,20 @@ void BraveNewTabMessageHandler::HandleSaveNewTabPagePref(
     LOG(ERROR) << "Invalid input";
     return;
   }
-  p3a::RecordValueIfGreater<NTPCustomizeUsage>(
-      NTPCustomizeUsage::kOpenedAndEdited, "Brave.NTP.CustomizeUsageStatus",
-      kNTPCustomizeUsageStatus, g_browser_process->local_state());
   PrefService* prefs = profile_->GetPrefs();
   // Collect args
   std::string settingsKeyInput = args[0].GetString();
   auto settingsValue = args[1].Clone();
   std::string settingsKey;
 
+  if (settingsKeyInput != "showToday" &&
```

Comment:
could you please add a comment regarding why do we exclude these settings as discussed in DM?